### PR TITLE
add keyboard shortcut to close child windows and add a custom menu to the child windows

### LINKED ISSUE #3510: common system shortcuts: ⌘W should close secondary windows
while Ctrl+Q resp. ⌘Q works as expected and exit the app,

Ctrl+W resp. ⌘W does not close window as expected.

one can argue that the main window should not be closed that way, however, for secondary windows, this shortcut is expected to work, eg. closing the help, a html-message-view or a webxdc-editor window - just all "real" windows that are not the main window

this is a very minor issue, feel free to close it if it gets complicated - but in case it is an easy thing, it'd be good to have it

### PR DESCRIPTION
**Pull request by Simon-Laux**
closes #3510

### CODE REVIEW

File: src/main/windows/help.ts [hunk 1]
```diff
@@ -115,4 +117,94 @@ export async function openHelpWindow(locale: string, anchor?: string) {
...
+    ])
+  }
+
+  if (!isMac) {
+    win.setMenu(makeMenu())
+  }
+
+  win.on('focus', () => {
+    if (isMac) {
+      // change to webxdc menu
```
Comment (WofWca) [inline src/main/windows/help.ts:200]: Not "webxdc"

File: src/main/windows/help.ts [hunk 2]
```diff
@@ -115,4 +117,94 @@ export async function openHelpWindow(locale: string, anchor?: string) {
...
+
+    return Menu.buildFromTemplate([
+      ...(isMac ? appMenu : []),
+      {
+        label: tx('global_menu_file_desktop'),
+        submenu: [
+          {
+            label: tx('close_window'),
+            click: () => {
+              window.close()
```
Comment (WofWca) [inline src/main/windows/help.ts:148]:
```suggestion
              win.close()
```

**Review by WofWca — CHANGES_REQUESTED** [review]
I'd DRY or add a comment that it's copy-pasted so that whoever changes the code in the future doesn't forget to change all 3 places. But it's not critical I guess, unlike the `win` vs `window` comment.

File: src/main/windows/help.ts [hunk 3]
```diff
@@ -115,4 +117,94 @@ export async function openHelpWindow(locale: string, anchor?: string) {
...
+    ])
+  }
+
+  if (!isMac) {
+    win.setMenu(makeMenu())
+  }
+
+  win.on('focus', () => {
+    if (isMac) {
+      // change to webxdc menu
```
Comment (Simon-Laux) [inline src/main/windows/help.ts:200]:
```suggestion
      // change to help menu
```

File: src/main/windows/html_email.ts [hunk 4]
```diff
@@ -124,6 +127,98 @@ export function openHtmlEmailWindow(
...
+    ])
+  }
+
+  if (!isMac) {
+    window.setMenu(makeMenu())
+  }
+
+  window.on('focus', () => {
+    if (isMac) {
+      // change to webxdc menu
```
Comment (WofWca) [inline src/main/windows/html_email.ts:211]:
```suggestion
      // change to email menu
```

**Review by WofWca — APPROVED** [review]
Not a big expert on menus, but it works.

**Pull request merged**